.ci/scripts/dswx_s1/dswx_s1_final_1.0_runconfig.yaml

```diff
@@ -167,9 +170,9 @@ RunConfig:
             flag_collapse_wtr_classes: True
             exclude_inundated_vegetation: False
             set_not_water_to_nodata: False
-            set_hand_mask_to_nodata: False
-            set_layover_shadow_to_nodata: False
+            set_hand_mask_to_nodata: True
+            set_layover_shadow_to_nodata: True
             set_ocean_masked_to_nodata: False
             save_tif_to_output: True
 
-          log_file: /home/dswx_user/output_dir/dswx-s1.log
+          log_file:
```

Comment:
Just pointing out there is no path here.

Reply:
This was intentional, as we do not want this log file to be validated as an output product